card/db/migrate/20150220141031_create_deck_routes_file.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+
+class CreateDeckRoutesFile < ActiveRecord::Migration
+  def up
+    routes_path = File.join Wagn.root, 'config', 'routes.rb'
+    if !File.exist? routes_path
+      template_path = File.join Wagn.gem_root, 'lib', 'wagn', 'generators', 'wagn', 'templates', 'config', 'routes.erb'
+      @include_jasmine_engine = false
+      decko_path = ''
+      routes = ERB.new(File.read template_path).result binding
+      File.open(routes_path, "w") { |f|  f.write routes }
+    end
+  end
+end
```

Comment:
mentioned this in call, but I don't think file management belongs in migrations.  that would mean, eg, you could copy a migrated db to an installation without a routes.rb file, and it would then be broken and there wouldn't be a clear mechanism for fixing it.

If we were sure we wanted routes.rb files in app decks, we would probably be better off with a rake task that creates a routes.rb file when it doesn't exist.  Then we could have `wagn update` run that task.

HOWEVER, I think I would suggest this, instead:
1. for now, app decks should use the old pattern: routes.rb in gem.
2. deck apps, on the other hand, will need it generated.
3. if app deck managers want to override this, they can add something like this to their application.rb
````
config.paths.add 'config/routes', :with=>'config/routes.rb'
````
We could also add a method like `use_local_routes` as a shortcut.